src/panoramic/cli/config/auth.py

```diff
@@ -34,3 +35,8 @@ def get_client_secret() -> str:
             return config['auth']['client_secret']
         # Backwards compatibility for auth credentials
         return config['client_secret']
+
+
+@functools.lru_cache()
+def get_dbt_profiles() -> Optional[Dict[str, Any]]:
+    return read_config().get('dbt', {}).get('profiles')
```

Comment:
Nit: you could use `read_config('dbt').get('profiles')`

Reply:
Done :+1: